src/App/Pages/Settings/OptionsPageViewModel.cs

```diff
@@ -58,6 +59,12 @@ public OptionsPageViewModel()
                 new KeyValuePair<string, string>("black", AppResources.Black),
                 new KeyValuePair<string, string>("nord", "Nord"),
             };
+            AutoDarkThemeOptions = new List<KeyValuePair<string, string>>
+            {
+                new KeyValuePair<string, string>("dark", AppResources.Dark),
+                new KeyValuePair<string, string>("black", AppResources.Black),
+                new KeyValuePair<string, string>("nord", "Nord"),
+            };
```

Comment:
I think we should use `Tuple` instead of `KeyValuePair` [here](https://gist.github.com/ljw1004/61bc96700d0b03c17cf83dbb51437a69) you have a gist with performance comparison. There are some caveats regarding allocation where `KeyValuePair` is better but in this case we don't care much given that there are just a few items.
It doesn't matter much with these few items but I also think it's more readable with Tuples.
What do you think?